cypress/e2e/utils.js

```diff
@@ -59,6 +69,30 @@ export function startUp(pageName) {
   cy.reload()
 }
 
+export function verifyTrackerDialog() {
+  cy.get('#automated-usage-tracking-tool-dialog').should('be.visible')
+  cy.get('#automated-usage-tracking-tool-dialog-content').should('contain.text', trackingToolText)
+  cy.get('#automated-usage-tracking-tool-dialog-confirm-button').click()
+
+  cy.intercept('POST', 'https://accounts.undefined.gigya.com/accounts.initRegistration', {
+    statusCode: 200,
+    body: {
+      callId: 'mockCallId',
+      errorCode: 0,
+      apiVersion: 2,
+      statusCode: 200,
+      statusReason: 'OK',
+      time: Date.now(),
+      regToken: 'mockRegToken',
+    },
+  })
+
+  cy.intercept('POST', 'https://accounts.undefined.gigya.com/accounts.setAccountInfo', {
+    statusCode: 200,
+    body: { callId: 'mockCallId', errorCode: 0, apiVersion: 2, statusCode: 200, statusReason: 'OK', time: Date.now() },
+  })
```

Comment:
Try to mock the tracker function instead of the endpoint.
In cypress we use the cy.stub to mock a function it functions like a spy, we use a stup on the startUp to mock the function that would open the popUp -> cy.stub(window, 'open').as('windowOpenStub')